### ISSUE TITLE
No require-double-brackets warning

### ISSUE THREAD

**Issue by EmilySeville7cfg** [koalaman/shellcheck#2344]
#### For bugs
- Rule Id (if any, e.g. SC1000): require-double-brackets
- My shellcheck version (`shellcheck --version` or "online"): 0.7.0
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

Run shellcheck for:

```sh
#!/usr/bin/env bash

[ -e /etc/issue ]

```

as following: `shellcheck -o require-double-brackets test.sh`

#### Here's what shellcheck currently says:
Nothing reported. 

#### Here's what I wanted or expected to see:

require-double-brackets warning expected.

**Comment by koalaman** [comment]
This warning was added after 0.7.2 and is not yet in a stable release. Please try again after the next release, 0.8.0!
